src/dbos-executor.ts

```diff
@@ -98,10 +98,20 @@ export const dbosNull: DBOSNull = {};
 
 /* Interface for DBOS configuration */
 export interface DBOSConfig {
-  poolConfig: PoolConfig;
+  // Public fields
+  name?: string;
+  readonly database_url?: string;
   readonly userDbclient?: UserDatabaseName;
+  readonly sysDbName?: string;
+  readonly logLevel?: string;
+  readonly otlpTracesEndpoints?: string[];
```

Comment:
Maybe add otlpLogsEndpoints to make it the same as Python.

Reply:
I'm leaving this to the upcoming OTLP work